dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProjectMapper.xml

```diff
@@ -180,4 +180,10 @@
             union select id as project_id  from t_ds_project where user_id=#{userId})
         </if>
     </select>
+
+    <select id="queryAllProjectForDependent" resultType="org.apache.dolphinscheduler.dao.entity.Project">
+        select
+        <include refid="baseSql"/>
+        from t_ds_project
```

Comment:
It is recommended to query only the required fields, such as name and code.

Reply:
fixed dependent node only gets item code and name, confirmed ui only gets code and name